common/src/autogluon/common/utils/deprecated_utils.py

```diff
@@ -27,7 +27,7 @@ def _deprecation_warning(
         warnings.warn(
             f"Deprecation Warning: {msg}. This will raise an error in the future!",
             category=DeprecationWarning,
-            stacklevel=2,
+            stacklevel=3,
         )
```

Comment:
@yinweisu an even "safer" option (more visible, but also more intrusive) would be to use `category=UserWarning`, which will ensure that the warnings are shown under the default warning filter configuration. Is that an overkill?